CLI.md

```diff
@@ -91,24 +91,24 @@ Options:
   -h, --help                              display help for command
 
 Commands:
+  # ARIO Network
```

Comment:
previously I was just updating this list from the help command 
<img width="865" alt="image" src="https://github.com/user-attachments/assets/0d5ee086-d1a2-4a1e-9891-cd436d7db089" />

Reply:
i'll organize these to match - the readme

Reply:
[4d410f0](https://github.com/ar-io/ar-io-sdk/pull/470/commits/4d410f096d8dedcd2e3c7cd275486f81216120f9)